src/components/Footer.js

```diff
@@ -1,18 +1,13 @@
-import {
-  Link,
-  Paper,
-  Typography
-} from "@mui/material";
+import { Paper, Typography } from "@mui/material";
+import theme from "../theme";
```

Comment:
is importing theme the correct way to do things? This seems like a hack
https://mui.com/customization/theming/

Reply:
Gradient was removed from header and footer